art/attacks/evasion/projected_gradient_descent/__init__.py

```diff
@@ -0,0 +1,7 @@
+from art.attacks.evasion.projected_gradient_descent.projected_gradient_descent import ProjectedGradientDescent
```

Comment:
I think we should leave this file empty and use internally the full import paths instead.